### PR TITLE
Adding more skills to the Lead and Principle Engieering functions

### PR DESCRIPTION
**Pull request by harperjb**
inspired by, good tech-lead bad tech-lead article from Foursquare. http://engineering.foursquare.com/2014/01/30/good-tech-lead-bad-tech-lead/

### CODE REVIEW

File: eng.md [hunk 1]
```diff
@@ -72,15 +72,15 @@ Well-versed in multiple key product components; able to define and contribute to
 
 ### Teaching and Mentoring
 
-Defines, documents, and evolves best practices for development in their team's area of focus; shepherds and aids in development of new projects across the organization; provides training and mentoring for multiple team members
+Defines, documents, and evolves best practices for development in their team's area of focus; shepherds and aids in development of new projects across the organization; provides training and mentoring for multiple team members; constently working to improve process, remove roadblocks, and broaden the technical capabilities of their team
```
Comment (gmcquillan) [inline eng.md]: s/constently/constantly/

Comment (harperjb) [inline eng.md]: Thanks!!!

File: eng.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ Helps to continuously maintain and improve quality across the stack; performs re
 
 ### Customer Success
 
-Understands customer needs and motivations; understands how engineering projects contribute to business goals
+Understands customer needs and motivations; understands how engineering projects contribute to business goals; embrases itterative product development
```
Comment (leclark) [inline eng.md]: s/embrases/embraces s/itterative/iterative

**Comment by leclark** [comment]
:+1:

**Comment by theetruscan** [comment]
:shipit:

**Pull request merged**